app/src/atoms/buttons/OnDeviceDisplay/RadioButton.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react'
+import styled, { css } from 'styled-components'
+import {
+  TYPOGRAPHY,
+  COLORS,
+  SPACING,
+  BORDERS,
+  Flex,
+} from '@opentrons/components'
+import { StyledText } from '../../text'
+
+import type { StyleProps } from '@opentrons/components'
+
+interface RadioButtonProps extends StyleProps {
+  buttonLabel: string
+  buttonValue: string | number
+  onChange: React.ChangeEventHandler<HTMLInputElement>
+  disabled?: boolean
+  isSelected?: boolean
+  size?: 'large' | 'small'
+}
+
+export function RadioButton(props: RadioButtonProps): JSX.Element {
+  const {
+    buttonLabel,
+    buttonValue,
+    disabled = false,
+    isSelected = false,
+    onChange,
+    size,
+  } = props
+
+  const isLarge = size === 'large'
+
+  const SettingButton = styled.input`
+    display: none;
+  `
+
+  const availableButtonStyles = css`
+    background: ${COLORS.mediumBlueEnabled};
+
+    &:active {
+      background-color: #94afd4;
+    }
+  `
+
+  const selectedButtonStyles = css`
+    background: ${COLORS.blueEnabled};
+    color: ${COLORS.white};
+
+    &:active {
+      background-color: #045dd0;
+    }
+  `
+
+  const disabledButtonStyles = css`
+    background-color: ${COLORS.darkBlack_twenty};
+    color: ${COLORS.darkBlack_sixty};
+    cursor: not-allowed;
+  `
+
+  // TODO: (ew, 2023-04-21): button is not tabbable, so focus state
+  // is not possible on ODD. It's testable in storybook but not in real life.
+  const SettingButtonLabel = styled.label`
+    border-radius: ${BORDERS.size_four};
+    cursor: pointer;
+    padding: ${isLarge ? SPACING.spacing5 : SPACING.spacingM};
+    width: 100%;
+
+    ${isSelected ? selectedButtonStyles : availableButtonStyles}
+    ${disabled && disabledButtonStyles}
+
+    &:focus-visible {
+      box-shadow: 0 0 0 ${SPACING.spacingS} ${COLORS.fundamentalsFocus};
```

Comment:
Just noticed that 👇 
https://www.figma.com/file/OIdG64Q5cgvEw82ish5Eee/Design-System-(Flex)?node-id=1224-61431&t=40X6Wfql6XIyqtAl-0

```ts
export const ODD_FOCUS_VISIBLE = `0 0 0 ${SPACING.spacing1} ${COLORS.fundamentalsFocus}`
```
But still not sure that why Mel sees the different visual 🤔 

```suggestion
      box-shadow: 0 0 0 ${SPACING.spacing2} ${COLORS.fundamentalsFocus};
```

Reply:
Oh that's odd. All the button focus borders in figma are 4px, not 2px. I'll change the constant to 0.25rem and use that.

Reply:
thanks.
But still not sure why Mel saw the difference between SmallButton and CardButton.
Actually I was tunneling my localhost:6060 via ngrok since s3 has a cache issue.

Reply:
Pretty sure she was looking at MediumButton, not SmallButton. It had the issue she was seeing, and I fixed that last night.